source/help/messaging/sending-messages.rst

```diff
@@ -1,39 +1,46 @@
 .. _sending-messages:
 
-Sending Messages
+Messages
 ================
+ 
+Sending messages and replying to messages are important ways to keep conversations active with your team. You can edit and delete messages after you have sent them. You can also generate and share links to any message in Mattermost.
 
-Messages are composed in the center pane and reply messages are composed in the right-hand sidebar. Depending on your system configuration, you can edit and delete messages after they are sent. You can also generate and share links to any message in Mattermost.
-
-Sending a Message
+Sending Messages
 -----------------
 
-Write a message by typing into the text input box, then press **ENTER** to send it. Use **SHIFT+ENTER** to create a new line without sending a message. To send messages by pressing **CTRL+ENTER**, go to **Main Menu > Account Settings > Send messages on Ctrl + Enter**.
+Write a message by typing into the text input box, then press ENTER to send it. Use SHIFT+ENTER to create a new line without sending a message. To send messages by pressing CTRL+ENTER, go to **Main Menu > Account Settings > Advanced > Send messages on Ctrl + Enter > Edit**.
 
-Replying to a Message
+Replying to Messages
 ---------------------
 
-Reply to messages to keep conversations organized in threads. Click the reply icon next to any message text to open the right-hand sidebar where you can compose and send your reply. Replies are indented slightly in the center pane to indicate that they are child messages of a parent message.
+Click the **[...]** link next to a message, then click **Edit**. This keeps conversations organized in threads.
```

Comment:
This describes how to edit a message, but the title for this section is "Replying to Messages"